Assets/HoloToolkit/Utilities/Scripts/AtlasReferenceUpdater/Editor/Extensions/SpriteAtlasExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.U2D;
+
+namespace HoloToolkit.Unity
+{
+    public static class SpriteAtlasExtensions
+    {
+        public static string SpritePackables = "m_EditorData.packables";
```

Comment:
This value is from Unity's serialization?
Should we make this const?

Reply:
Oops yes, it was supposed to be const.